lms/djangoapps/certificates/management/commands/cert_allowlist_generation.py

```diff
@@ -93,3 +94,13 @@ def handle(self, *args, **options):
                         course=course_key
                     ))
                 generate_allowlist_certificate_task(user, course_key)
+
+
+def _get_user_from_identifier(identifier):
```

Comment:
I think your PR may be out of date with master. This command doesn't call this anymore.

Reply:
got it